internal/sponge/item/model.go

```diff
@@ -27,10 +28,12 @@ const sourceIDField = "id"
 // are formatted with an ID, Type, and Version, and asssociated data (which
 // may differ greatly between items) is encoded into the Data interface.
 type Item struct {
-	ID      string                 `bson:"item_id" json:"item_id" validate:"required,min=1"`
-	Type    string                 `bson:"type" json:"type" validate:"required,min=2"`
-	Version int                    `bson:"version" json:"version" validate:"required,min=1"`
-	Data    map[string]interface{} `bson:"data" json:"data"`
+	ID        string                 `bson:"item_id" json:"item_id" validate:"required,min=1"`
+	Type      string                 `bson:"type" json:"type" validate:"required,min=2"`
+	Version   int                    `bson:"version" json:"version" validate:"required,min=1"`
+	Data      map[string]interface{} `bson:"data" json:"data"`
+	CreatedAt time.Time              `bson:"created_at" json:"created_at"`
+	UpdatedAt time.Time              `bson:"updated_at,omitempty" json:"updated_at,omitempty"`
```

Comment:
I don't think that the `omitempty` should be there. If a document is created, it was also "updated" at that time, so for each `Item`, we always have a `created_at` and `updated_at` field available.